frontend/src/components/Exercise/ExerciseRuntimeTab.tsx

```diff
@@ -1,25 +1,20 @@
-import { Button, Container } from 'react-bootstrap'
-import React, { useEffect } from 'react'
-
 import Editor from "@monaco-editor/react";
-import { pushNotification } from "../../features/notification/notificationSlice";
-import { useDispatch } from 'react-redux';
 import { useState } from 'react'
 import { useUpdateExerciseMutation } from "../../features/courses/exerciseApiSlice";
+import { ArrowPathIcon } from '@heroicons/react/24/solid';
+
 
 const ExerciseRuntimeTab = (props: any) => {
 
     const { course, session, exercise, isOwner } = props
 
     const [prefix, setPrefix] = useState("");
     const [suffix, setSuffix] = useState("");
-    const dispatch = useDispatch();
     const [updateExercise] = useUpdateExerciseMutation();
+    const [notificationMessage, setNotificationMessage] = useState("");
+    const [notificationType, setNotificationType] = useState("");
```

Comment:
There exists a reusable notification system, used like :

```
dispatch(pushNotification({
    message: "...",
    type: "success" / "error" / ...
})
```

It has been updated, the types used to correspond to Bootstrap colors.

We can keep using it.

Reply:
Done